back/engines/commercial/multi_tenancy/lib/tasks/core/templates.rake

```diff
@@ -82,8 +82,19 @@ namespace :templates do
       exit(1)
     end
 
-    release_prefix = MultiTenancy::Templates::Utils.new.release_templates
+    template_utils = MultiTenancy::Templates::Utils.new
+    release_prefix = template_utils.release_templates
     puts({ event: 'templates_release', status: 'success', release_prefix: release_prefix }.to_json)
+
+    # Remove files of the previous release (the new test prefix). We do this to ensure
+    # that:
+    # - the next release won't be polluted by files from the previous releases
+    # - templates that no longer exist are removed
+    counts = template_utils.clear_test_templates
+    if counts[:errors_count].positive?
+      puts({ event: 'templates_cleanup', status: 'failed', counts: counts }.to_json)
```

Comment:
I'd have a preference for using `Rails.logger.info` here in terms of keeping the logs consistent - or actually `.error` would be more appropriate here

Reply:
Will put on my TODO.